### DISCUSSION TITLE
Azure DataFactory cant connect

### DISCUSSION THREAD

**Discussion by DinoBenciniSBS** [community/community#66015] [New to GitHub]
I removed my key for Azure DataFactory in GitHub Settings and now i cant get it back or connect from Azure Datafactory to a repository in my GitHub. Please Help?

**Comment by queenofcorgis** [comment]
Hi there @DinoBenciniSBS 👋🏼 

Would you mind going into your account settings under applications and confirming that Azure DataFactory is an authorized application?

**Reply by DinoBenciniSBS** [reply]
No there,s nothing there, how would i add it?

**Reply by queenofcorgis** [reply]
I'm not super familiar with connecting the two beyond the basics, but this may help: https://learn.microsoft.com/en-us/azure/data-factory/connector-github?tabs=data-factory
